app/src/main/java/com/github/se/studybuddies/database/DatabaseConnection.kt

```diff
@@ -708,13 +708,13 @@ class DatabaseConnection {
     }
   }
 
-  // using the topicData collection
+  // using the topicData and topicItemData collections
   suspend fun getTopic(uid: String): Topic {
     val document = topicDataCollection.document(uid).get().await()
     return if (document.exists()) {
-      val name = document.getString("name") ?: ""
-      val exercisesList = document.get("exercises") as List<String>
-      val theoryList = document.get("theory") as List<String>
+      val name = document.getString(Companion.topic_name) ?: ""
```

Comment:
Nice modification !
To nitpick, could you either put everywhere Companion.xxx or remove the Companion here ?